torchsparse/nn/modules/conv.py

```diff
@@ -193,3 +203,47 @@ def forward(self, inputs: SparseTensor):
         outputs = outputs.view(batch_size, *self.bev_shape, -1)
         outputs = outputs.permute(0, 3, 1, 2).contiguous()
         return outputs
+
+
+class ToBEVHeightCompression(nn.Module):
```

Comment:
It would be helpful to write a short description for each `ToBEV` function to help users understand the difference between them.